gapstool.vbs

```diff
@@ -306,14 +313,70 @@ Sub RunDiagnostics(CompName)
   RunCommand "dir ""\\" & CompName & "\c$\Program Files (x86)\Google\Google Apps Password Sync"" /B /S", _
              "instx86"
 
-  PrintLine "Geting system-wide proxy settings dump from registry - proxy.txt"
+  PrintLine "Getting system-wide proxy settings dump from registry - proxy.txt"
   RunCommand "reg query ""\\" & CompName & "\HKLM\SOFTWARE\Policies\Microsoft\Windows\CurrentVersion\Internet Settings"" /v ProxySettingsPerUser", _
              "proxy"
 
   PrintLine "Getting system-wide WinHTTP settings dump from registry - winhttp.txt"
   RunCommand "reg query ""\\" & CompName & "\HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Connections"" /v WinHttpSettings", _
              "winhttp"
 
+  PrintLine "Getting system-wide WinHTTP settings dump from registry, and decoding - winhttp_decoded.txt"
+  ' Create a WMI StdRegProv.
+  Dim objStdRegProv
+  Set objStdRegProv = GetObject("winmgmts:{impersonationLevel=impersonate}!\\" & _
+                                CompName & "\root\default:StdRegProv")
+  PrintErrorIfNeeded "Error opening WMI StdRegProv on " & CompName & ": "
+  ' Retrieve the value of WinHTTPSettings from the registry.
+  ' Note that GetBinaryValue returns an array, where each element in the array
+  ' is a DECIMAL value of the octets. ¯\_(ツ)_/¯

```

Comment:
Please avoid Unicode characters, I'm not sure what effects they'll have on weird old systems. :)